src/class-fields/class-private-field-whitespace-syntax-error.case

```diff
@@ -0,0 +1,10 @@
+// Copyright (C) 2018 Bloomberg LP. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+ /*---
+desc: Syntax error if whitespace after '#' sigil
+template: syntax/invalid
+features: [class-fields-private]
+---*/
+
+//- fields
+# x = 1
```

Comment:
we already have some name convention for these errors in this folder: `grammar-privatename-*.case`. It's also covered in here:

https://github.com/tc39/test262/blob/master/src/class-fields/grammar-privatename-error.case

It's probably good to follow the same convention from the filename and some text from info tag used there (showing at least the `# IdentifierName` parts).